lib/nanoc/cli/commands/view.rb

```diff
@@ -52,6 +52,10 @@ def run
         run Rack::File.new(site.config[:output_dir])
       end.to_app
 
+      # Print a link
+      url = "http://#{options[:host] || 'localhost'}:#{options_for_rack[:Port]}/"
```

Comment:
You could use `options_for_rack[:host]` instead of `options[:host] || 'localhost'`.

This made me wonder whether defaulting to 0.0.0.0, rather than 127.0.0.1, is a sensible default—it probably isn’t. :/